tuf_conformance/test_keys.py

```diff
@@ -159,17 +156,17 @@ def test_keytype_and_scheme(
     signer = repo.new_signer(keytype, scheme)
     repo.add_key(Root.type, signer=signer)
     repo.root.roles[Root.type].threshold += 1
-    repo.bump_root_by_one()
+    repo.publish([Root.type], bump_version=True)
 
     assert client.refresh(init_data) == 0
     assert client.version(Root.type) == 2
 
     # Create new root version. Replace the correct signature with one that looks
     # reasonable for the keytype but is incorrect. Expect client to refuse the new root.
-    repo.bump_root_by_one()
-    root_md = Metadata.from_bytes(repo.signed_roots.pop())
+    repo.publish([Root.type], bump_version=True)
```

Comment:
don't need to specify `bump_version=True`

Reply:
https://github.com/theupdateframework/tuf-conformance/pull/188/commits/487b3882ec4107beece4d9ca2364f3dc8a17a5f4